src/backup.c

```diff
@@ -1136,7 +1145,7 @@ close_file(io_write_proxy_t *fd)
 
 	int res = io_proxy_close2(fd, FILE_PROXY_EOF);
 	if (res != 0) {
-		err("Error while closing backup file");
+		err("Error while closing backup io proxy");
```

Comment:
Isn't this almost always a file unless backing up to S3?

Reply:
Since this function deals in io proxies, I changed the err text here. This error message was duplicated throughout  the code, I wanted to make more of them unique for easier debugging.